bin/all_sky_search/pycbc_coinc_findtrigs

```diff
@@ -127,6 +128,13 @@ for i in range(len(args.trigger_files)):
                             args.gating_veto_windows)
     ifo = reader.ifo
     trigs.ifos.append(ifo)
+    
+    # We don't have that many triggers compared to num templates so
+    # likely many zeros which can be pre-culled
+    if len(reader.file[ifo]['template_id']) < num_templates * 10:
```

Comment:
the *10 factor feels hacky .. it doesn't really tell us if there are lots of unoccupied templates. I think numpy.unique should be fairly fast, so why not always do that and decide based on the actual number of different templates?  Or if unique/intersect1s aren't always fast enough, decide based on the length of the template_id dataset (ie whatever determines if it will be slow)?

Reply:
Well, the implicit assumption I guess is that triggers aren't say all clustered in a single template. I guess that could be wrong. Ok, I've switched to  a fixed threshold